ddtrace/tracer/tracer_metadata.go

```diff
@@ -4,14 +4,14 @@
 // Copyright 2023 Datadog, Inc.
 package tracer
 
-// TracerMetadata represents the configuration of the tracer.
+// Metadata represents the configuration of the tracer.
 //
 //go:generate msgp -unexported -marshal=true -o=tracer_metadata_msgp.go -tests=false
-type TracerMetadata struct {
```

Comment:
I believe this is a new change that hasn't been released yet. The name change should be fine.